ci/Jenkinsfile.linux-nix

```diff
@@ -6,7 +6,7 @@ def isPRBuild = utils.isPRBuild()
 def isNightlyBuild = utils.isNightlyBuild()
 
 pipeline {
-  agent { label 'linux && x86_64 && nix-2.24' }
+  agent { label 'linux && x86_64 && nix-2.24 && go-1.22' }
```

Comment:
Why is this tag necessary? Isn't Nix providing Go?

Reply:
my bad.